src/libs/ship/src/ship_lights.h

```diff
@@ -82,6 +82,7 @@ class ShipLights : public IShipLights
         bool bVisible;
         float fTotalBrokenTime, fBrokenTime;
         bool bOff, bBrokenTimeOff;
+		bool bTmpOff;
```

Comment:
The naming is counter-intuitive. could you please rename it to something meaningful...